src/main/resources/i18n/messages.properties

```diff
@@ -155,23 +155,30 @@ FoldTab.39=Ignore articles "a" and "the" while sorting
 FoldTab.40=Cancel scanning shared folders
 FoldTab.41=Canceling scan...
 FoldTab.43=Prettify filenames (does not rename the actual files)
-FoldTab.44=<html><strong>Default:</strong> Enabled<br><strong>Notes:</strong> If enabled, files will be ordered like this:<ol><li>Apple</li><li>The Banana</li><li>Carrot</li><li>A Date</li></ol>If disabled, files will be ordered like this:<ol><li>Apple</li><li>A Date</li><li>Carrot</li><li>The Banana</li></ol></html>
-FoldTab.45=<html><strong>Default:</strong> Disabled<br><strong>Notes:</strong> If enabled, files will appear like this:<ol><li>Apple - 10</li><li>Banana - 213 - The Cake Special</li><li>Carrot (2002)</li></ol>If disabled, files will appear like this:<ol><li>[SomeFans] Apple - 10 (1280x720 Blu-Ray FLAC) [44A36D20]</li><li>Banana.S02E13.The.Cake.Special.720p.HDTV.x264-SomeFans</li><li>Carrot.2002.LIMITED.720p.BluRay.x264-SomeFans</li></ol></html>
-FoldTab.47=<html><strong>Default:</strong> Disabled<br><strong>Notes:</strong> If enabled, 3 new virtual folders will be displayed on your device/s to allow you to access your iTunes music:<ul><li>Browse by Artist</li><li>Browse by Album</li><li>Browse by Genre</li></ul></html>
-FoldTab.49=<html><strong>Default:</strong> 10000<br><strong>Notes:</strong> If the number of items in a folder exceeds this number, they will be divided into virtual subfolders by their first letter/number</html>
+FoldTab.44=Enabled
+FoldTab.44Note=If enabled, files will be ordered like this:<ol><li>Apple</li><li>The Banana</li><li>Carrot</li><li>A Date</li></ol>If disabled, files will be ordered like this:<ol><li>Apple</li><li>A Date</li><li>Carrot</li><li>The Banana</li></ol>
+FoldTab.45=Disabled
```

Comment:
These defaults should not be in the translations file at all - that is what the story is meant to be about. We should automatically show the default.

Reply:
@SubJunk we need to translate `Enabled` and `Disabled`. What do you mean to a make it automatically? Do you mean to read the UMS.conf and look for the default value stored in it?

Reply:
My understanding was also that `Enabled` and `Disabled` had to be translated. 

I think there were two expectations of what the story needed. If we could all align before I make anymore changes that would be great.

Reply:
Yes "Enabled" and "Disabled" need to have one entry each in this file, and that can be reused.
So for booleans we detect whether the default value is `true` or `false`, and then display the correct value: if the default is `true`, display the translation for `Enabled`, if it is false, display the translation for `Disabled`.
Does that clarify it?

Reply:
Thanks for that @SubJunk Where would I be able to see the boolean value for the default? As an example would I check the `configuration.isUseSymlinksTargetFile()` for the default of `useSymlinksTargetFile`?

Reply:
The goal is to not repeat the translalation for values and make it only once.

Reply:
Yes you could start a fresh `configuration` instance so it is all defaults (since the existing one will return the values the user has changed too) and then do what you suggested